docs/docs-content/vm-management/environment-setup.md

```diff
@@ -12,25 +12,44 @@ resources.
+
+### Control Plane Nodes
 
 Typically, the cluster control plane nodes do not operate any VMO workloads. As a result, they can have lighter hardware
-specifications. For example, a server with 4 cores and 8 GB RAM is sufficient for a minimum-specification control plane
-node.
+specifications in terms of CPU and RAM. For example, a server with 4 cores and 8 GB RAM is sufficient for a
+minimum-specification control plane node. The rest of the hardware requirements for control plane nodes remain the same
+as worker nodes.
+
+| Component            | Minimum                                                                                            | Recommended                                        | Comments                                                                                             |
+| -------------------- | -------------------------------------------------------------------------------------------------- | -------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| **Form Factor**      | The server must fit Fiber Channel (FC) adapters and have sufficient Network Interface Cards (NIC). | 2U Rackmount Chassis                               |                                                                                                      |
+| **Network Adapters** | 2 x 10 Gbps <br /> (data + management)                                                             | 2 x 10 Gbps (data) <br /> 2 x 10 Gbps (management) | Pod overlay operates on the management network.                                                      |
+| **Storage Adapters** | 2 x 16 Gbps FC                                                                                     | 2 x 16 Gbps FC                                     | Storage adapters must support the FC protocol, a high-speed network protocol used for data transfer. |
+| **Disks**            | Local disk for the OS boot (SAN boot is supported)                                                 | Local disk for the OS boot                         | Boot from SAN requires special consideration due to the multi-path configuration.                    |
 
-You can increase the hardware specifications based on the total number of control plane and worker nodes you want in the
-cluster. Refer to the following table for guidance on control plane node sizing. Note that these recommendations assume
-that each cluster has at least three control plane nodes.
+The CPU and RAM requirements for control plane nodes increase with the number of worker nodes and namespaces in the
+cluster. Refer to the following table for guidance on control plane node sizing.
+
+::: warning
```

Comment:
@lennessyy I'm not sure why this is happening. I wonder if the space between the last `:` and the word `warning` is causing this 🤔 

![CleanShot 2024-07-23 at 08 21 05](https://github.com/user-attachments/assets/33ac257f-237a-4763-8227-80145e3204b4)